### PR TITLE
Links to the Dev Portal for external devs

### PR DESCRIPTION
**Pull request by cfranceschi-ledger**
### 📝 Description

I previously added links to the Dev Portal and Discord, where external devs might use this repository. 
I updated the link to Discord and added the links in the readmes of LedgerJS packages.

### ❓ Context

- **Impacted projects**: `` External developers (Blockchain Support / Ledger support in third-party wallets)

### ✅ Checklist

- [ ] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 📸 Demo

No demo

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: apps/ledger-live-desktop/README.md [hunk 1]
```diff
@@ -209,6 +209,6 @@ This part of the repository is where you will add the support of your blockchain
 For a smooth and quick integration:
 
 - See the developers’ documentation on the [Developer Portal](https://developers.ledger.com/docs/coin/general-process/) and
-- Go on Discord to chat with developer support and the developer community. See you there! If you are new to Ledger OP3N Discord server [click here](https://discord.gg/Ledger), otherwise directly join [the Blockchain channel](https://discord.com/channels/885256081289379850/907623688759803935).
+- Go on [Discord]((https://developers.ledger.com/discord-pro/)) to chat with developper support and the developper community.
```
Comment (hedi-edelbloute) [inline apps/ledger-live-desktop/README.md:212]: I think there is a single "p" on developer ! (to apply on other changes too)

Comment (cfranceschi-ledger) [reply apps/ledger-live-desktop/README.md]: Yes, you're right! I've corrected everything.

**Review by hedi-edelbloute — APPROVED** [review]

**Comment by gre** [comment]
Hey, I'm sorry, i have merged this one too fast thinking it's just README, but actually some of these files are self generating and are not passing CI.

@cfranceschi-ledger 
i've reverted the change on develop, we'll have to redo the PR, the doc must be placed BEFORE the `# API` which is the part that is generated.

File: libs/ledgerjs/packages/hw-app-algorand/README.md [hunk 2]
```diff
@@ -51,3 +51,18 @@ cosmos.getAddress("44'/283'/0'/0/0").then(o => o.address)
 ```
 
 Returns **[Promise](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Promise)<{publicKey: [string](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String), address: [string](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String)}>** an object with a publicKey, address and (optionally) chainCode
+
+
+---
```
Comment (gre) [inline libs/ledgerjs/packages/hw-app-algorand/README.md:56]: that block must be placed BEFORE the `# API`, otherwise it gets rewritten

Comment (cfranceschi-ledger) [reply libs/ledgerjs/packages/hw-app-algorand/README.md:56]: OK! No problem, I will redo the PR.

**Pull request merged**